chia/clvm/singleton.py

```diff
@@ -1,3 +1,5 @@
+# Package: obsolete
```

Comment:
presumably we keep this singleton around because we still want the wallet to be able to identify old and obsolete singletons. Creating a package for everything that's obsolete seems quite odd though. For example, the wallet will depend on `obsolete` because the old singleton is in there. And `condition_tools.py` belongs to `obsolete` because of functions that we'd like to get rid of in there. `condition_tools.py` will necessarily depend on a bunch of lower level functionality. Now all of a sudden, the wallet "pulls in" all the condition dependencies. I think making a project for all obsolete code is a mistake and will make it harder to untangle dependencies.

I think we should just accept that every package may have some obsolete stuff in it that we'll want to remove over time.

Reply:
Yeah this is not an opinion that an `obsolete` package should exist just like the `todo` package should not exist.  It's more so avoiding forming an opinion about placing it in some arbitrary package and instead marking it for deletion.  I didn't handle the deletion of such files in this PR in order to remain focused.

As to why this file is obsolete, any files that need these puzzles can use the clvm import functionality that this file uses on its own. It's a pointless file because it can be easily circumvented and probably should be.